pkg/init/Dockerfile

```diff
@@ -1,4 +1,5 @@
 FROM linuxkit/alpine:bc528cf9d4065d2e09aa44ff76909b94cfe8d867 AS build
+
```

Comment:
seems like an unrelated/accidental change.

Reply:
I have not the foggiest clue how this got in here. Pretty funny. Fixing now. Thanks for catching @rn

Reply:
Fixed.